src/AVFoundation/AVAssetTrack.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.Versioning;
+using Foundation;
+using CoreMedia;
+
+
+namespace AVFoundation {
+	public partial class AVAssetTrack {
+
+		/// <summary>
+		/// An array of <see cref="T:CoreMedia.CMFormatDescription" />s that describe the formats of the samples in the <see cref="T:AVFoundation.AVAssetTrack" />.
+		/// </summary>
+		/// <value>To be added.</value>
+		/// <remarks>To be added.</remarks>
+		public CMFormatDescription [] FormatDescriptions {
+			get {
+				return (Array.ConvertAll (FormatDescriptionsAsObjects,
+					obj => {
+						var description = CMFormatDescription.Create (obj.Handle, false);
+						GC.KeepAlive (obj);
+						return description;
+					}
+				) as CoreMedia.CMFormatDescription [])!;
```

Comment:
Array.ConvertAll, so I don't think you need the cast?

If not, you can make it explicit:

```suggestion
				return Array.ConvertAll<NSObject, CMFormatDescription (FormatDescriptionsAsObjects,
					obj => {
						var description = CMFormatDescription.Create (obj.Handle, false);
						GC.KeepAlive (obj);
						return description;
					}
				);
```

Reply:
This is what the original code was doing, I tried to change the least amount of code possible.